data/data-pipeline/data_pipeline/etl/score/etl_utils.py

```diff
@@ -108,3 +110,174 @@ def floor_series(series: pd.Series, number_of_decimals: int) -> pd.Series:
     )
 
     return floored_series
+
+
+def _create_df_from_yaml_contents(
+    fields_list_from_yaml: list,
+    fields_to_store_in_codebook: list,
+) -> pd.DataFrame:
+    """Helper function to create a dataframe from yaml fields
+
+    This function does:
+        1. Creates a dictionary that will eventually become a dataframe.
+           This dictionary has columns dictated by whichever fields
+           the yaml file has that we want to keep, specified by fields_to_store.
+        2. Goes through the fields and adds each type of field to the dictionary. If
+           the field is missing, appends a null value.
+        3. Returns a dataframe that includes all of the fields specified and
+           also is indexed by the column name that is native to the CEJST team
```

Comment:
"the column name that is native to the CEJST team" - this feels like a very imprecise way of saying what you're trying to say?

Reply:
updated